### PR TITLE
fix(ios): TabGroup bar is wrongly transparent on iOS 15

### PR DESCRIPTION
**Pull request by jquick-axway**
**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-28505
https://jira.appcelerator.org/browse/TIMOB-28523

**Summary:**
- On iOS 15, the bottom tab bar in a tab group is completely transparent.
  * Should be translucent by default and show the color of tab window.
  * Prevented "tabsBackgroundColor" property from changing the bar color.
- Breaking-change on iOS 15 set "appearance" to transparent by default. Also requires us to set "scrollEdgeAppearance".

**Tab Bar Translucency Test:**
1. Create a Classic app project from template. (Provides tab bar icons.)
2. Use `TabGroupColorWindowTest.js` attached to [TIMOB-28505](https://jira.appcelerator.org/browse/TIMOB-28505) for the "app.js".
3. Build and run on iOS 15.
4. Verify bottom bar shows 3 tab titles/icons and is green as shown in the iOS 14 screenshot in [TIMOB-28505](https://jira.appcelerator.org/browse/TIMOB-28505).
5. Tap on tab 2 and verify the "red" window shows through the bottom tab bar.
6. Tap on tab 3 and verify the "blue" window shows through the bottom tab bar.

**Tab Bar Color Test:**
1. Create a Classic app project from template. (Provides tab bar icons.)
2. Use `TabGroupColorSharedTest.js` attached to [TIMOB-28505](https://jira.appcelerator.org/browse/TIMOB-28505) for the "app.js".
3. Build and run on iOS 15.
4. Verify bottom tab bar's background is light-green.
5. Tap on tab 2 and 3 and verify tab bar's background is still light-green.

### CODE REVIEW

**Comment by jquick-axway** [comment]
Closing since Apple fixed this issue as of Xcode 13 beta 3.

**Comment by jquick-axway** [comment]
Re-opening this PR since the issue came back when testing it with Xcode 13 beta 5.

File: iphone/Classes/TiUITabGroup.m [hunk 1]
```diff
@@ -42,6 +42,12 @@ - (UITabBarController *)tabController
     controller.delegate = self;
     controller.moreNavigationController.delegate = self;
     [TiUtils configureController:controller withObject:self.proxy];
+    if ([TiUtils isIOSVersionOrGreater:@"15.0"]) {
```
Comment (ewanharris) [inline iphone/Classes/TiUITabGroup.m:45]: I _think_ this section will need wrapping in a `#if IS_SDK_IOS_15` block. I believe that's the cause of the build failure in the tests on Jenkins

File: iphone/Classes/TiUITabGroup.m [hunk 2]
```diff
@@ -365,6 +371,10 @@ - (void)setTabsBackgroundColor_:(id)value
   UITabBar *tabBar = [controller tabBar];
   //A nil tintColor is fine, too.
   [tabBar setBarTintColor:[color color]];
+  if ([TiUtils isIOSVersionOrGreater:@"15.0"]) {
```
Comment (ewanharris) [inline iphone/Classes/TiUITabGroup.m:374]: Same here for a `#if IS_SDK_IOS_15` block.

**Review by garymathews — APPROVED** [review]
CR: **PASS**

**Review by ewanharris — APPROVED** [review]
FR Pass. TabGroup bar is now displayed as it was on iOS 14

**Pull request merged**